torch/csrc/api/src/nn/modules/activation.cpp

```diff
@@ -0,0 +1,24 @@
+#include <torch/nn/modules/activation.h>
+#include <torch/nn/functional/activation.h>
+
+namespace F = torch::nn::functional;
+
+namespace torch {
+namespace nn {
+
+ELUImpl::ELUImpl(const ELUOptions& options_) : options(options_) {}
+
+Tensor ELUImpl::forward(const Tensor& input) {
+  return F::elu(input, options);
+}
+
+void ELUImpl::reset() {}
+
+void ELUImpl::pretty_print(std::ostream& stream) const {
+  stream << std::boolalpha
+         << "torch::nn::ELU(alpha=" << options.alpha()
+         << ", inplace=" << options.inplace() << ")";
```

Comment:
In the Python version, it seems that whether it prints `inplace` is conditional:
```python
    def extra_repr(self):
        inplace_str = ', inplace=True' if self.inplace else ''
        return 'alpha={}{}'.format(self.alpha, inplace_str)
```
and I think we should do the same here.